e2e/cypress/integration/menus/menu_actions_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @menu
+
+describe('LHS Menu action', () => {
+    before(() => {
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        cy.apiCreateTeam('team', 'Team');
+    });
+
+    it('MM-T1669 Team icon shows active state on click', () => {
+        // # Assert if the team wrapper in LHS present
+        cy.get('.team-wrapper').should('be.visible').within(() => {
+            // # Assert if the teams bar present
+            cy.get('[data-rbd-droppable-id="my_teams"]').should('have.length', 1).within(() => {
+                // # Assert more than one teams are present
+                cy.get('.team-container').should('have.length.gt', 1);
+
+                // # Assert only one team is in active state
+                cy.get('.active').should('have.length', 1);
+
+                // # move the first team by clicking
+                cy.get('.team-container').first().click();
```

Comment:
Since team container is scrollable and first element may be out of view,

```suggestion
                cy.get('.team-container').first().scrollIntoView().click();
```